mypyc/test-data/run-classes.test

```diff
@@ -467,6 +467,21 @@ a = A(10)
 assert a.foo() == 11
 assert foo() == 21
 
+[case testClassKwargs]
+class X:
+    def __init__(self, msg: str, **variables: int) -> None:
+         pass
+[file driver.py]
+import traceback
+from native import X
+X('hello', a=0)
+try:
+    X('hello', msg='hello')
+except TypeError as e:
+    print(f"{type(e).__name__}: {e}")
+[out]
```

Comment:
```suggestion
    traceback.print_exc()
[out]
Traceback (most recent call last):
  File "driver.py", line 5, in <module>
    X('hello', msg='hello')
[out version>=3.11]
Traceback (most recent call last):
  File "driver.py", line 5, in <module>
    X('hello', msg='hello')
    ~^^^^^^^^^^^^^^^^^^^^^^
```

The other tests all use conditional `out`. _Not sure which version added the `^^^^` here though, could be `3.11` or maybe `3.13`._